src/VisualStudio/CSharp/Impl/PackageRegistration.pkgdef

```diff
@@ -198,4 +198,4 @@
 [$RootKey$\SettingsManifests\{13c3bbb4-f18f-4111-9f54-a0fb010d9194}]
 @="Microsoft.VisualStudio.LanguageServices.CSharp.LanguageService.CSharpPackage"
 "ManifestPath"="$PackageFolder$\UnifiedSettings\csharpSettings.registration.json"
-"CacheTag"=qword:3AEFDB0DA1308654
+"CacheTag"=qword:18C11F0A543B8AD0
```

Comment:
what is this?

Reply:
When you add/modify properties in `csharpSettings.registration.json`, you need to change this key to make sure the registration file take effect. Also It is useful if you want to deploy and test locally (like devenv /rootsuffix RoslynDev), everytime you change the value, VS would reload the C# page.

The tag value we used is generated by xxHash with the input of csharpSettings.registration.json, so it's also covered by unit test. When someone forget to modify it, test will fail